### PR TITLE
Bump up version for nuget release for Azure Management Network

### PR DESCRIPTION
**Pull request by ritwikbasu**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-    <VersionPrefix>12.0.0-preview</VersionPrefix>
+    <VersionPrefix>12.0.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @ritwikbasu remove this and replace here with < version > but don't keep both tags.
Also move the < version > tag to the top property group and not the propertygroup below.

Also you need to bump up the assembly version in AssemblyInfo.cs

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj (hunk 1 above)
Comment (ritwikbasu) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Resolved in commit update

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("12.0.0.0")]
-[assembly: AssemblyFileVersion("12.0.0.0")]
+[assembly: AssemblyVersion("12.0.1.0")]
+[assembly: AssemblyFileVersion("12.0.1.0")]
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: @ritwikbasu can you revert the file version, you don't need to update minor version updates to your file version.
Will create bunch of issue when you have dot releases

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (ritwikbasu) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: Resolved

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**